buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -889,9 +883,6 @@
     "DockerTag_Ubuntu1604": {
       "value": "ubuntu-16.04-debpkg-e5cf912-20174703024721"
     },
-    "DistroRid_Ubuntu1604": {
-      "value": "ubuntu.16.04-$(PB_TargetArchitecture)"
```

Comment:
Maybe I've missed it but where are these distro specific values getting set now?

Reply:
They are being inferred from the current machine.

`RuntimeEnvironment.GetRuntimeIdentifier()` will return `ubutnu.16.04-x64` and that gets put into `$(HostMachineRid)`.  and by default `$(OutputRid)` is set to `$(HostMachineRid)-$(TargetArchitecture)`.

Reply:
Note that all these steps do the same loop:

* Set up docker for a specific distro
* Build the installer for that specific distro
* Publish the installer for that specific distro

Since the build and publish are happening on `ubuntu.16.04` machines, `$(HostMachineRid)` just works.

Reply:
great.

Reply:
can we call $(MachineRid) instead of $(HostMachineRid) ? It might be confusing in case of host machine running docker container for building installer. 
For e.g. It can be a Ubuntu14.04 host machine that runs Ubuntu16.04 container to build the installers. In this case, $(HostMachineRid) will be ubuntu.16.04-x64 which is confusing as it may infer Ubuntu.14.04-x64 per the naming convention.

Reply:
The intention of the property is the RID of the current machine that is running the build.  Just calling it `MachineRid` can also be confused with "this is RID of the machine that we are building for".

For what it's worth, the dotnet/cli calls this property [`HostRid`](https://github.com/dotnet/cli/blob/master/build/HostInfo.targets#L15).  One intention here is to try to start using the same name for the same thing across repos.